garrysmod/lua/includes/modules/draw.lua

```diff
@@ -299,6 +299,25 @@ function TexturedQuad( tab )
 
 end
 
+--[[---------------------------------------------------------
+	Name: Circle( x, y, radius, segments )
+	Desc: pawrapper
+-----------------------------------------------------------]]
+function Circle( x, y, radius, seg )
+	local cir = {}
+
+	table.insert( cir, { x = x, y = y, u = 0.5, v = 0.5 } )

```

Comment:
Instead of calling table.insert here, make it the first key of the table during creation.